lib/improver/cli/standardise.py

```diff
@@ -32,181 +32,41 @@
-        default=25000., type=float,
-        help=("Radius of vicinity to search for a coastline, in metres. "
-              "Default value; 25000 m"))
-
-    regrid_group.add_argument(
-        "--regridded_title", metavar="REGRIDDED_TITLE", default=None, type=str,
-        help="New title to be used for the regridded field.")
-
-    # metadata standardisation
-    parser.add_argument("--fix_float64", action='store_true', default=False,
-                        help="Check and fix cube for float64 data. Without "
-                             "this option an exception will be raised if "
-                             "float64 data is found but no fix applied.")
-    parser.add_argument("--json_file", metavar="JSON_FILE", default=None,
-                        help='Filename for the json file containing required '
-                             'changes that will be applied '
-                             'to the attributes. Defaults to None.')
-    parser.add_argument("--coords_to_remove", metavar="COORDS_TO_REMOVE",
-                        nargs="+", type=str, default=None,
-                        help="List of names of scalar coordinates to be "
-                             "removed from the non-standard input.")
-    parser.add_argument("--new_name", metavar="NEW_NAME", type=str,
-                        default=None, help="New dataset name.")
-    parser.add_argument("--new_units", metavar="NEW_UNITS", type=str,
-                        default=None, help="Units to convert to.")
-
-    args = parser.parse_args(args=argv)
-
-    if args.target_grid_filepath or args.json_file or args.fix_float64:
-        if not args.output_filepath:
-            msg = ("An argument has been specified that requires an output "
-                   "filepath but none has been provided")
-            raise ValueError(msg)
-
-    if (args.input_landmask_filepath and
-            "nearest-with-mask" not in args.regrid_mode):
-        msg = ("Land-mask file supplied without appropriate regrid_mode. "
-               "Use --regrid_mode=nearest-with-mask.")
-        raise ValueError(msg)
-
-    if args.input_landmask_filepath and not args.target_grid_filepath:
-        msg = ("Cannot specify input_landmask_filepath without "
-               "target_grid_filepath")
-        raise ValueError(msg)
-
-    # Load Cube and json
-    attributes_dict = load_json_or_none(args.json_file)
-    # source file data path is a mandatory argument
-    output_data = load_cube(args.source_data_filepath)
-    target_grid = None
-    source_landsea = None
-    if args.target_grid_filepath:
-        target_grid = load_cube(args.target_grid_filepath)
-        if args.regrid_mode in ["nearest-with-mask"]:
-            if not args.input_landmask_filepath:
-                msg = ("An argument has been specified that requires an input "
-                       "landmask filepath but none has been provided")
-                raise ValueError(msg)
```

Comment:
I think this error check has not been replicated because it is already in the `plugin.__init__` method. 👍

Reply:
Yes.